samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Disable serial and UART interface.
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+CONFIG_LOG=n
+
+# RAM usage configuration
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# BT configuration
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=1
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=n
+CONFIG_BT_BUF_ACL_RX_SIZE=502
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+CONFIG_BT_CTLR_PHY_2M=n
+
+# 802.15.4 configuration
+CONFIG_NRF_802154_SER_RADIO=y
+CONFIG_NRF_802154_ENCRYPTION=y
+CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=2
+
+# Debug and assert configuration
+CONFIG_ASSERT=y
+CONFIG_DEBUG_INFO=y
+CONFIG_EXCEPTION_STACK_TRACE=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+CONFIG_REBOOT=n
+
+# IPC
+CONFIG_MBOX=y
```

Comment:
I think this `MBOX` config is no longer needed. There will be a warning about it if I remember correctly.

Reply:
fixed.